voltdbclient/url_test.go

```diff
@@ -23,28 +23,52 @@ func TestParseURL(t *testing.T) {
 	}
 
 	for _, s := range sample {
-		u, err := parseURL(s.conn)
-		if s.pass {
-			if err != nil {
-				t.Fatal(err)
-			}
-			if u.Scheme != s.scheme {
-				t.Errorf("expected %s got %s", s.scheme, u.Scheme)
-			}
-			if u.User.Username() != s.user {
-				t.Errorf("expected %s got %s", s.user, u.User)
-			}
-			p, _ := u.User.Password()
-			if p != s.passwd {
-				t.Errorf("expected %s got %s", s.passwd, p)
-			}
-			if u.Host != s.host {
-				t.Errorf("expected %s got %s", s.host, u.Host)
-			}
-		} else {
-			if err == nil {
-				t.Fatal("expected an error")
+		s := s //capture variable
+		t.Run("", func(t *testing.T) {
```

Comment:
its used for best test practices. I use it as habits and to see the all errors if happens more than one. Also its still useless unless we don't need to tear up/down something.
I'm not the fun the parallel testing but its more useful and consistent for whole system.
If you prefer to use table-driven test without sub-tests, i can remove it. Won't be a problem in this case.